.package create: /workdir/PyAudio-0.2.12/.tox/.package
.package installdeps: setuptools>=61.0, wheel
warning: check: missing meta-data: if 'author' supplied, 'author_email' should be supplied too

cpythonlibtest-unit-test-tests create: /workdir/PyAudio-0.2.12/.tox/cpythonlibtest-unit-test-tests
cpythonlibtest-unit-test-tests installdeps: mock, pytest
cpythonlibtest-unit-test-tests inst: /workdir/PyAudio-0.2.12/.tox/.tmp/package/1/PyAudio-0.2.12.tar.gz
cpythonlibtest-unit-test-tests installed: exceptiongroup==1.2.0,iniconfig==2.0.0,mock==5.1.0,packaging==23.2,pluggy==1.3.0,PyAudio @ file:///workdir/PyAudio-0.2.12/.tox/.tmp/package/1/PyAudio-0.2.12.tar.gz#sha256=e17a63478a0a790568e2ab5afe064ad3c9f9ff4f377107342b1e4939c0bd6254,pytest==7.4.3,tomli==2.0.1
cpythonlibtest-unit-test-tests run-test-pre: PYTHONHASHSEED='3608013062'
cpythonlibtest-unit-test-tests run-test: commands[0] | pytest -v --tb=native --junitxml /workdir/results/PyAudio/0.2.12/1/cpython-test-results.xml
============================= test session starts ==============================
platform linux -- Python 3.10.8, pytest-7.4.3, pluggy-1.3.0 -- /workdir/PyAudio-0.2.12/.tox/cpythonlibtest-unit-test-tests/bin/python
cachedir: .tox/cpythonlibtest-unit-test-tests/.pytest_cache
rootdir: /workdir/PyAudio-0.2.12
collecting ... collected 0 items

- generated xml file: /workdir/results/PyAudio/0.2.12/1/cpython-test-results.xml -
============================ no tests ran in 0.01s =============================
ERROR: InvocationError for command /workdir/PyAudio-0.2.12/.tox/cpythonlibtest-unit-test-tests/bin/pytest -v --tb=native --junitxml /workdir/results/PyAudio/0.2.12/1/cpython-test-results.xml (exited with code 5)
___________________________________ summary ____________________________________
ERROR:   cpythonlibtest-unit-test-tests: commands failed
